proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,1047 @@
+
+* [ ] [`article`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) is equivalent to `View` with `display:block` layout and `role="article"`.
+* [ ] [`aside`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).
+* [ ] [`div`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div) is equivalent to `View` with `display:block` layout.
+* [ ] [`footer`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer).
+* [ ] [`h1-6`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).
+* [ ] [`header`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).
+* [ ] [`main`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main).
+* [ ] [`nav`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).
+* [ ] [`section`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) is equivalent to `View` with `display:block` layout and `role="section"`.
+
+## Text
+
+Text content elements organize blocks or sections of content. Important for accessibility, these elements identify the purpose or structure of that content.
+
+* [ ] [`blockquote`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote).
+* [ ] [`li`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li).
+* [ ] [`ol`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol).
+* [ ] [`ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).
+* [ ] [`p`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p).
+* [ ] [`pre`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre).
+
+## Inline text
+
+Inline text semantics define the meaning, structure, or style of a word, line, or any arbitrary piece of text.
+
+* [ ] [`a`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
+* [ ] [`b`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b).
+* [ ] [`bdi`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi).
+* [ ] [`bdo`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdo).
+* [ ] [`code`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code).
+* [ ] [`em`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em).
+* [ ] [`span`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span) is equivalent to `Text` with `display:inline` layout.
+* [ ] [`strong`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong).
+* [ ] [`sub`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sub).
+* [ ] [`sup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/sup).
+
+## Media
+
+* [ ] [`img`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) is similar to `Image` without advanced loader configuration.
```

Comment:
Implementing `picture` with fallbacks to different supported sources would be great, especially since there's a high probability that a custom runtime may not support a the same formats across platforms. e.g. `.heic` on iOS with fallback to `jpg` on web.

Reply:
Maybe at some point. But `picture` has very little use inside Meta, and isn't commonly used outside Meta from what I can see.